Configuration/TypoScript/PluginHaiku/constants.typoscript

```diff
@@ -0,0 +1,12 @@
+
+plugin.tx_examples_haiku {
+    view {
+        templateRootPath = EXT:examples/Resources/Private/Layouts
+        partialRootPath = EXT:examples/Resources/Private/Partials
+        layoutRootPath = EXT:examples/Resources/Private/Templates
+    }
+    settings {
+        singlePid = 0
```

Comment:
Why "0" and not just empty?

```
singlePid =
```

Reply:
cause this is an integer?

Reply:
It is TypoScript, so everything is basically a string ;-) But it does not matter.